### PR TITLE
Log cleanup

### PR DESCRIPTION
**Pull request by s-n-d-p**
[x] removed InvokeHostname
[x] removed FailureInfo
[x] removed Url
[x] removed VideoSrcUrl, VideoDataOriginalSrcUrl
[x] removed BookmarkInfo
[x] removed Id
[x] removed PageLanguage

### CODE REVIEW

**Comment by s-n-d-p** [comment]
@DhruvM7 FYI - Removed `PageLanguage` tag as IMO it doesn't provide much, if any, value addition to the logs.

File: src/scripts/clipperUI/clipper.tsx [hunk 1]
```diff
@@ -176,8 +176,6 @@ class ClipperClass extends ComponentBase<ClipperState, {}> {
 				this.captureBookmarkContent();
 
 				Clipper.logger.setContextProperty(Log.Context.Custom.ContentType, OneNoteApi.ContentType[updatedPageInfo.contentType]);
-				Clipper.logger.setContextProperty(Log.Context.Custom.InvokeHostname, UrlUtils.getHostname(updatedPageInfo.rawUrl));
-				Clipper.logger.setContextProperty(Log.Context.Custom.PageLanguage, updatedPageInfo.contentLocale);
```
Comment (ksboora) [inline src/scripts/clipperUI/clipper.tsx:180]: why do we need to remove page language and locale info?

File: src/scripts/extensions/pageNavInject.ts [hunk 2]
```diff
@@ -107,7 +107,5 @@ export class PageNavInject extends FrameInjectBase<FrameInjectOptions> {
 	 */
 	private setPageInfoContextProperties(): void {
 		this.logger.setContextProperty(Log.Context.Custom.ContentType, OneNoteApi.ContentType[DomUtils.getPageContentType(document)]);
-		this.logger.setContextProperty(Log.Context.Custom.InvokeHostname, UrlUtils.getHostname(document.URL));
-		this.logger.setContextProperty(Log.Context.Custom.PageLanguage, DomUtils.getLocale(document));
```
Comment (ksboora) [inline src/scripts/extensions/pageNavInject.ts:111]: same here.

File: src/scripts/clipperUI/clipper.tsx (hunk 1 above)
Comment (s-n-d-p) [reply src/scripts/clipperUI/clipper.tsx:180]: Pl. see https://github.com/OneNoteDev/WebClipper/pull/499#issuecomment-1025527965
Sometimes `link` HTML tag is creeping into `PageLanguage` and that contains an `href` attribute containing the kind of data we are looking to cleanup.

**Review by ksboora — APPROVED** [review]

File: src/scripts/extensions/clipperInject.ts [hunk 3]
```diff
@@ -192,7 +192,7 @@ export class ClipperInject extends FrameInjectBase<ClipperInjectOptions> {
 			label: Log.Failure.Label.UnhandledExceptionThrown,
 			properties: {
 				failureType: Log.Failure.Type.Unexpected,
-				failureInfo: { error: JSON.stringify({ error: e.toString(), url: window.location.href }) },
```
Comment (susinh) [inline src/scripts/extensions/clipperInject.ts:195]: Should we remove his JSON structure from the failureInfo? We might have dashboards/scripts that rely on the Failure info having the specified structure

File: src/scripts/extensions/webExtensionBase/webExtensionWorker.ts [hunk 4]
```diff
@@ -93,7 +93,7 @@ export class WebExtensionWorker extends ExtensionWorkerBase<W3CTab, number> {
 						label: Log.Failure.Label.UnclippablePage,
 						properties: {
 							failureType: Log.Failure.Type.Expected,
-							failureInfo: { error: JSON.stringify({ error: WebExtension.browser.runtime.lastError.message, url: this.tab.url }) },
```
Comment (susinh) [inline src/scripts/extensions/webExtensionBase/webExtensionWorker.ts:96]: Should we remove his JSON structure from the failureInfo? Do we have any dashboards/scripts that rely on the Failure info having the specified structure

File: src/scripts/contentCapture/pdfScreenshotHelper.ts [hunk 5]
```diff
@@ -36,6 +36,7 @@ export class PdfScreenshotHelper {
 
 			let errorCallback = (failureInfo: OneNoteApi.RequestError) => {
 				getBinaryEvent.setStatus(Log.Status.Failed);
+				failureInfo.response = failureInfo.responseHeaders = undefined;
```
Comment (susinh) [inline src/scripts/contentCapture/pdfScreenshotHelper.ts:39]: Just confirming. Here, we intend to set both response and responseHeaders as undefined. Right?

File: src/scripts/extensions/webExtensionBase/webExtensionWorker.ts (hunk 4 above)
Comment (s-n-d-p) [reply src/scripts/extensions/webExtensionBase/webExtensionWorker.ts:96]: The key part (as in key-value) of the structure is same.  For the value part, earlier we were stringifying an entire object, which contained the keys - `error` and `url`. The requirement was to remove `url`.
To maintain the same format we can set url to '' (empty string), but I avoided that in favor of cleaner code. I'll reach out to you offline to discuss further.

File: src/scripts/contentCapture/pdfScreenshotHelper.ts (hunk 5 above)
Comment (s-n-d-p) [reply src/scripts/contentCapture/pdfScreenshotHelper.ts:39]: Yes.

File: src/scripts/extensions/webExtensionBase/webExtensionWorker.ts (hunk 4 above)
Comment (s-n-d-p) [reply src/scripts/extensions/webExtensionBase/webExtensionWorker.ts:96]: No existing dashboard relies on FailureInfo.

**Review by susinh — APPROVED** [review]

**Pull request merged**